src/panels/lovelace/cards/hui-markdown-card.ts

```diff
@@ -53,12 +76,43 @@ export class HuiMarkdownCard extends LitElement implements LovelaceCard {
           class="markdown ${classMap({
             "no-header": !this._config.title,
           })}"
-          .content="${this._config.content}"
+          .content="${this._content}"
         ></ha-markdown>
       </ha-card>
     `;
   }
 
+  private async _connect() {
+    if (!this._unsubRenderTemplate && this._hass && this._config) {
+      this._unsubRenderTemplate = subscribeRenderTemplate(
+        this._hass.connection,
+        (result) => {
+          this._content = result;
+        },
+        {
+          template: this._config.content,
+          entity_ids: this._config.entity_id,
+        }
+      );
+      this._unsubRenderTemplate.catch(() => {
+        this._content = this._config!.content;
+      });
+    }
+  }
+
+  private async _disconnect() {
+    if (this._unsubRenderTemplate) {
+      this._unsubRenderTemplate
+        .then((unsub) => {
+          this._unsubRenderTemplate = undefined;
+          return unsub();
```

Comment:
why return it ?

Reply:
Because it returns a promise, which the `catch` below applies to.

Reply:
In that case, I prefer an async function and await it. It creates better stack traces and more clear for the reader of the code to see what is going on.

Reply:
Ah yes. Brainfart.

I must say I'm impressed by your patience sometimes.